onnxruntime/contrib_ops/cuda/math/cufft_plan_cache.h

```diff
@@ -36,7 +36,7 @@ struct ParamsHash {
   // Params must be a POD because we read out its memory
   // contenst as char* when hashing
 
-  static_assert(std::is_pod<T>::value, "Params is not POD");
+  static_assert(std::is_trivial<T>::value, "Params is not POD");
```

Comment:
nit: can we also update the static_assert message? e.g., "x is not a trivial type".

also can update the comment in the previous line, e.g. "x must be a trivial type".

Reply:
+1 Everywhere